shell/platform/android/io/flutter/embedding/engine/android/AndroidTouchProcessor.java

```diff
@@ -15,26 +17,46 @@
 public class AndroidTouchProcessor {
 
   // Must match the PointerChange enum in pointer.dart.
-  // TODO(mattcarroll): convert these to an IntDef
-  private static final int POINTER_CHANGE_CANCEL = 0;
-  private static final int POINTER_CHANGE_ADD = 1;
-  private static final int POINTER_CHANGE_REMOVE = 2;
-  private static final int POINTER_CHANGE_HOVER = 3;
-  private static final int POINTER_CHANGE_DOWN = 4;
-  private static final int POINTER_CHANGE_MOVE = 5;
-  private static final int POINTER_CHANGE_UP = 6;
+  @IntDef({
+      PointerChange.CANCEL,
+      PointerChange.ADD,
+      PointerChange.REMOVE,
+      PointerChange.HOVER,
+      PointerChange.DOWN,
+      PointerChange.MOVE,
+      PointerChange.UP
+  })
+  private @interface PointerChange {
+    int CANCEL = 0;
+    int ADD = 1;
+    int REMOVE = 2;
+    int HOVER = 3;
+    int DOWN = 4;
+    int MOVE = 5;
+    int UP = 6;
+  }
+
 
   // Must match the PointerDeviceKind enum in pointer.dart.
-  // TODO(mattcarroll): convert these to an IntDef
-  private static final int POINTER_DEVICE_KIND_TOUCH = 0;
-  private static final int POINTER_DEVICE_KIND_MOUSE = 1;
-  private static final int POINTER_DEVICE_KIND_STYLUS = 2;
-  private static final int POINTER_DEVICE_KIND_INVERTED_STYLUS = 3;
-  private static final int POINTER_DEVICE_KIND_UNKNOWN = 4;
+  @IntDef({
+      PointerDeviceKind.TOUCH,
+      PointerDeviceKind.MOUSE,
+      PointerDeviceKind.STYLUS,
+      PointerDeviceKind.INVERTED_STYLUS,
+      PointerDeviceKind.UNKNOWN
+  })
+  private @interface PointerDeviceKind {
+    int TOUCH = 0;
+    int MOUSE = 1;
+    int STYLUS = 2;
+    int INVERTED_STYLUS = 3;
+    int UNKNOWN = 4;
+  }
 
   // Must match the unpacking code in hooks.dart.
-  private static final int POINTER_DATA_FIELD_COUNT = 19;
-  private static final int BYTE_PER_FIELD = 8;
+  private static final int POINTER_DATA_FIELD_COUNT = 21;
```

Comment:
At some point this will need to be updated for the additional fields added as part of scroll wheel support

Reply:
Added a TODO